src/main/java/Controller/newPlaylistController.java

```diff
@@ -0,0 +1,36 @@
+package Controller;
+import InputBoundary.newPlaylistInputBoundary;
+import InputData.newPlaylistInputData;
+import UseCase.createPlaylist;
+import Presenter.playlistPresenter;
+
+/**
+ * Controller-layer implementation for create playlist use case
+ */
+public class newPlaylistController {
+    private final newPlaylistInputBoundary newPlaylistInputBoundary;
+    private final newPlaylistInputData newPlaylistInputData;
+    private createPlaylist createPlaylist;
+
+    public newPlaylistController(newPlaylistInputData newPlaylistInputData, playlistPresenter presenter){
+        this.createPlaylist = new createPlaylist(presenter);
+        this.newPlaylistInputBoundary = createPlaylist;
+        this.newPlaylistInputData = newPlaylistInputData;
+    }
+    /**
+     * @param presenter presenter object to store output message in
+     * @return String confirmation of the playlist's creation
+     */
+    public void createNewPlaylist(playlistPresenter presenter){
+        //TODO: Assuming this section functions like createPlaylist object w/ input data
+        // and setting the presenter's output message to the confirmation stored in aforementioned object
+        // which is accessed by showConfirmation()
+        this.newPlaylistInputBoundary.newPlaylist(this.newPlaylistInputData);
+        presenter.setOutputMessage(this.newPlaylistInputBoundary.showConfirmation());
```

Comment:
IIRC the diagram suggests the use case should take the presenter as a parameter (as an OutputBoundary) and call the presenter's function from within the use case interactor, but this technically does not break clean architecture. 

I'd prefer we keep the places where the presenter is called as consistent as possible, however.

Reply:
That method I changed in the most recent commit I'm going to push now, so it should be better.